src/lib.rs

```diff
@@ -233,6 +355,9 @@ impl<'a> Input<'a> {
 ///
 ///To reinterpret it as different type add `as <type>` where type can be: `u8`, `u16`, `u32`, `u64` or `u128`.
 ///
+///Change the endianness of how the file is interpreted by adding `le` or `be` suffix, e.g. `u16le` or `u32be`.
```

Comment:
Do you think this is clear now? Or should we add a bit more explanation?

Reply:
I would elaborate further by adding.

> You can use endianness suffix to change expected endianness within file. When endianness is specified macro shall treat integers within file correspondingly to suffix and converting it to target's native endianness. Allowed suffixes: `le` and `be`